### ISSUE TITLE
Convert not working

### ISSUE THREAD

**Issue by Arkobat** [JCulley/ProQuests-Documentation#224]
**Spigot account** - https://www.spigotmc.org/members/freakdk.749/
<br>

**Plugin version** - 2.1.1.2
<br>

**Spigot Version** - This server is running Paper version git-Paper-454 (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)
<br>

**Link to log file** - https://hastebin.com/kuguhupoha.md
<br>

**Addition Information**
The command **/quest convert <file>** doesn't work.
Found on https://www.spigotmc.org/resources/18249/update?update=122000

Trying to type: /quest convert mad1
https://gyazo.com/47e0b30500d4186d2bb724b970d8f9e5

**Comment by JCulley** [comment]
Hi,

Seems how you have been rather hostile to me in the past I ask you go and remove your offensive and unreasonable review you previously had posted before I am willing to offer the time out of my day to assist you.

Additionally I need a more thorough explanation please explain how the command “doesn’t work” as that is a very vague and open ended statement which I wouldn’t know where to begin investigating if a problem exists.

**Comment by JCulley** [comment]
Nevermind the update you linked was from 2016 which is no way applicable anymore, if you check the commands page you will see this command is not listed. This is because it was removed. Feel free to remove your review prior to asking for assistance in the future however .

**Comment by arkobat** [comment]
So... No way of updating.
Not interested in giving support?
No wonder you have so many bad reviews...
Just try to read what people are saying. If you wanted to help them, a lot of the bad reviews would be avoided and properly been better.

But why remove code.
If having a version that works (at least somehow, I've never had any experience with this working 100%, but you don't really care to help customers...) why should I update, and risking breaking something?

**Comment by freak-dk** [comment]
@JCulley 
This is my account, Arkobat is just dealing with the setup of a new server.

But are you being serious?
Obviously the profanity is a bit much but in no way is it directed at you, I am simply stating how you indeed "left me fucked" by a, in my eyes as a customer, very poor and inconsiderate decision. 

That review is, as you said, from 2 YEARS ago and nothing but a fair review on how you acted back then.
Causing me to have to delete a LOT of important data - Having to explain that to a community the size of mine was not easy and deffinitely warrented a negative review.

Now. 
If we get the plugin working again I will happily re-review it the plugin in its current state, but as of this moment it does not work for us. 
I have asked Arkobat to open up a new issue and supply relevant information.

The convert command thing isn't really relevant to our issue. We'll manage converting the files ourselves.

**Comment by JCulley** [comment]
> Having to explain that to a community the size of mine was not easy and deffinitely warrented a negative review.

It still doesn’t warrant posting such a negative review. You didn’t HAVE to update if your playerdata was such a concern the plugin worked perfectly fine however I wished to improve it further. When you purchase a plugin you are guaranteed one version, me supplying additional updates is something I do without receiving any extra value from you.

If it isn’t “really relevant” then don’t open an issue to begin with. I ask you do your own testing and investigation before opening issues which can easily be solved using documentation.

I’m not getting any further into this over a GitHub issue send me a PM if you would like to discuss this however as I have stated I will not be providing my time to assist someone who had taken their time to try and negatively attack me for making improvements just to ask for assistance with them in the future.

**Comment by freak-dk** [comment]
@JCulley 
I obviously, as I stated, did not open this Issue.
Arkobat just was not aware that this wasn't a part of the plugin anymore. I was, because I have owned this since close to the first release, but we had not spoken about this before.

I only updated because I had to. 
No matter how you put it, not offering a way to salvage data was a dick move. Because of the poor way you had data stored back then I was not able to develop a solution myself and that is simply stating a fact.
We had PLENTY of pm's back and forth through Spigot back then. 

I agree discussing this is futile but so is asking me to remove a perfectly warrented negative review before you will offer me any support.
If you feel it is against the Spigot TOS feel free to report, I personally do not think it is. If I did I would remove it.
